camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java

```diff
@@ -136,10 +137,17 @@ public void before() throws IOException, NoSuchFieldException, IllegalAccessExce
   }
 
   @After
-  public void after() throws IOException {
+  public void after() throws IOException, SecurityException, NoSuchFieldException,
+    IllegalArgumentException, IllegalAccessException {
     // Delete all camus data
     folder.delete();
-    _mocks.clear();
+    mocks.clear();
+    EtlInputFormatForUnitTest.consumerType = EtlInputFormatForUnitTest.ConsumerType.REGULAR;
+    EtlInputFormatForUnitTest.recordReaderClass = EtlInputFormatForUnitTest.RecordReaderClass.REGULAR;
+    EtlRecordReaderForUnitTest.decoderType = EtlRecordReaderForUnitTest.DecoderType.REGULAR;
+    Field field = EtlMultiOutputFormat.class.getDeclaredField("committer");
```

Comment:
After each test, reset all behavior to REGULAR, so that settings in the current test won't affect future tests.